mono/mini/interp/transform.c

```diff
@@ -52,12 +52,15 @@ typedef struct
 	unsigned char flags;
 } StackInfo;
 
+
+#define STACK_VALUE_NONE 0
+#define STACK_VALUE_LOCAL 1
```

Comment:
`enum`?

Reply:
I think I had some restrictions/warnings at some point with enums in c++ (in c they used to just be ints) so I stopped using enums forever for low level stuff.